config/locales/en.yml

```diff
@@ -607,7 +607,7 @@ en:
       why_public: Why are serial numbers and non-stolen bikes public?
       serial_numbers_public: We make serial numbers public because it's the best way to recover stolen bikes and deter theft.
       the_faster_and_easier_it_is: The faster and easier it is to access serials, the more stolen bikes we recover. Restricting access to serial numbers only helps thieves.
-      this_isnt_optimistic_speculation: 'This isn''t optimistic speculation: we''ve proved this and continue to do so with every stolen bike we recover.'
```

Comment:
My prettier just fixes stuff @jmromer!

Hopefully this doesn't break things for you

Reply:
No worries! There's a rake task form i18n-tasks we can use to format translation files, I'm just waiting for `en.yml` to be populated to run it.